File: program/print-reverse-of-a-number/PrintReverseOfANumber.swift
import Swift
import Foundation

print("Enter a number")
var number = Int(readLine()!)!
var reverseNum = 0

print("Original number: ", number)

while(number != 0){
   reverseNum = reverseNum * 10
   reverseNum = reverseNum + number % 10
   number = number/10
}
print("Reversed number: ", reverseNum)

File: program/program/print-reverse-of-a-number/PrintReverseOfANumber.kt
import java.util.Scanner

fun main() {
    print("Enter number : ")
    println()
    var min = readLine()

    println("Reversed Number: " + ReverseNumber(min!!.toInt()))

}

private fun ReverseNumber(num: Int): Int {
    var reversed = 0
    var num2= num

    while (num2 != 0) {
        val digit = num2 % 10
        reversed = reversed * 10 + digit
        num2 /= 10
    }

    return  reversed
}
